src/pages/home/report/ReportActionsList.tsx

```diff
@@ -211,6 +211,8 @@ function ReportActionsList({
         return ReportConnection.getReport(report.reportID)?.lastReadTime ?? report.lastReadTime ?? '';
     }, [report.reportID, report.lastReadTime]);
 
+    const lastReadTime = transactionThreadReport?.lastReadTime && transactionThreadReport.lastReadTime > reportLastReadTime ? transactionThreadReport.lastReadTime : reportLastReadTime;
```

Comment:
Please add a comment explaining why this is necessary

Reply:
Added

Reply:
Love it!